README.md

```diff
@@ -2507,6 +2507,21 @@ Get a general event from XSC matching the provided msi.
 event, err := xscManager.GetAnalyticsGeneralEvent(msi)
 ```
 
```

Comment:
Get analytics general event --> Get Analytics General Event
Update analytics general event --> Update Analytics General Event
Add analytics general event --> Add Analytics General Event
Report XSC analytics metrics --> Report XSC Analytics Metrics

The above changes should be made in both the table of contents and the sections titles.